src/tree/ContainerAppTreeItem.ts

```diff
@@ -91,32 +92,22 @@ export class ContainerAppTreeItem extends AzExtParentTreeItem implements IAzureR
         await openUrl(`https://${this.data.configuration?.ingress?.fqdn}`);
     }
 
-    public async deleteTreeItemImpl(context: IActionContext & { suppressPrompt?: boolean }): Promise<void> {
+    public async deleteTreeItemImpl(context: IActionContext & { suppressNotification?: boolean }): Promise<void> {
```

Comment:
Was this intentional?

Reply:
I made the assumption that `suppressPrompt` also meant `suppressNotification`.  I was referencing some other delete implementations and it looked like we also typically suppress showing output windows in the execute step.  Since we're checking `context.suppressNotification` in the execute step, I figured we needed to pass it that value when we initialized the `wizardContext` so that it aligns with this configuration (that way we can pull it by just doing `...context`):

![Screen Shot 2022-09-01 at 11 51 02 AM](https://user-images.githubusercontent.com/40250218/187990945-2af4a5a1-e671-4f8b-9903-f0b5bbf68da2.png)

Thoughts?

Reply:
`suppressNotification` is an activity log specific idea, which correlates to the user setting. This setting is on by default and makes it so that notifications don't appear if the same info could be gathered from the activity.

<img width="376" alt="image" src="https://user-images.githubusercontent.com/12476526/187993582-fa316bcd-c0cf-409d-b173-625adf1ae56e.png">

`suppressPrompt` is unrelated to `suppressMessage` and is just a flag that when `true`, means we shouldn't confirm prompt the user before completing the action. In this case that means we shouldn't prompt `DeleteConfirmationStep`.

Reply:
Okay, that makes sense, thanks for that clarification!

Reply:
We could use some naming improvements for sure.